pootle/static/js/shared/components/Modal.js

```diff
@@ -58,8 +82,14 @@ const Modal = React.createClass({
   /* Handlers */
 
   handleClose() {
+    if ('onCanClose' in this.props && !this.props.onCanClose()) {
```

Comment:
This one looks unused. Personally I find it hard to understand what it is even meant to do too.